packages/ide-extension/src/enhancement/enhancements.json

```diff
@@ -9,7 +9,8 @@
                 "exec": {
                     "extensionId": "sapse.sap-ux-application-modeler-extension",
                     "commandId": "sap.ux.environmentcheck.archiveProject"
-                }
+                },
+                "platforms": ["VSCODE", "BAS"]
```

Comment:
@unseen1980 Please change `platforms` to `environment`. That will sound better given that both are IDEs

Reply:
done!